source/atlas-data-lake-testing/tests/aggregate.yml

```diff
@@ -0,0 +1,16 @@
+tests:
```

Comment:
I think this comment was missed: https://github.com/mongodb/specifications/pull/821#discussion_r452606100

Please add:

```yaml
collection_name: &collection_name "driverdata"
database_name: &database_name "test"  
```

How was this passing in the Java driver? I doubt the CRUD spec test runner happens to use `test.driverdata` as its default namespace.

Reply:
Not sure how it passes in the Java driver. I have added the collection and database names.